src/Microsoft.TestPlatform.Build/Resources/Resources.Designer.cs

```diff
@@ -1,26 +1,25 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
+//     Version du runtime :4.0.30319.42000
```

Comment:
You need to undo these comment language changes. They should only be in English. You can update the localizations in XLF instead of putting it in source file

Reply:
Nice catch. Now I know why I was afraid of meddling with those resources. 😓

Reply:
Been there; done that!! 😇